api/apps/articles/model/schema.coffee

```diff
@@ -22,7 +22,7 @@ videoSection = (->
     name: 'Video'
     isTypeOf: (data) -> data.type is 'video'
   ).keys
-    type: @string().valid('video')
+    type: @string().valid('video', 'basic')
```

Comment:
@eessex - Not sure where the model save polling is coming from but this was the issue with saving `cover_image_url`

Reply:
Ah cool -- yes we do use a video section for the hero_section on classic articles, this makes sense to me for features.